openfold/utils/loss.py

```diff
@@ -504,6 +503,29 @@ def lddt_ca(
     all_atom_positions = all_atom_positions[..., ca_pos, :]
     all_atom_mask = all_atom_mask[..., ca_pos : (ca_pos + 1)]  # keep dim
 
+
+    return lddt(
+        all_atom_pred_pos,
+        all_atom_positions,
+        all_atom_mask,
+        cutoff=cutoff,
+        eps=eps,
+        per_residue=per_residue,
+    )
+
+def lddt_ca_multimer(
```

Comment:
If it uses the same way as lddt_ca : ```all_atom_mask[..., ca_pos : (ca_pos + 1)]```  then the dimension is not correct. I have to perform this ``` all_atom_mask[..., None] - all_atom_mask[..., None, :]``` so that ```all_atom_mask``` ends up with a shape of ```[batch_size,n_res,n_res] ```